### ISSUE TITLE
mitohifi adds second newline to input

### ISSUE THREAD

**Issue by YanisChrys** [marcelauliano/MitoHiFi#76]
Hello,
I am writing this to report that when using an already folder input fasta file, I assume when trying to fix the headers, the pipeline adds a second newline to each row. Not only is it not expected or good to directly edit the input, this can cause problem in many places down the line. Is it possible to remove this functionality?

(I am using the mitohifi container)

**Comment by jgnunes** [comment]
You are right, thanks for catching that. We'll fix this behaviour in the next release, which should be published soon. Once it is published I'll answer you here and close the issue.
